server/src/main/java/com/demo/Controller.java

```diff
@@ -33,6 +33,11 @@ public Controller(DaprClient client, MessageRepository messageRepository) {
     @ResponseStatus(code = HttpStatus.OK)
     public String health() {
         logger.debug("-- healthy --");
+        return "healthy";
+    }
+
+    @GetMapping("/generateId")
```

Comment:
According to REST standard, path should point to the resource, not the action. That's why it is unrecommended to use verbs in paths

Reply:
okey-dokey